flatland_viz/src/spawn_model_tool.cpp

```diff
@@ -349,7 +349,7 @@ void SpawnModelTool::LoadPolygonFootprint(
   }
   if (points.size() > 0) {
```

Comment:
Was this no longer needed? or never needed?).
If so, maybe we can just remove the whole thing.